### PR TITLE
W-10186674 - [OFM+] Installers: Update Power of Us link to Trailblazer community

### PR DESCRIPTION
**Pull request by scottwarren-sfdo**
[\[OFM+\] Installers: Update Power of Us link to Trailblazer community
](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000QUQCdYAP/view)

Replaced the Metadeploy installer PowerOfUs Hub links with Trailblazer Community links.

Old: https://powerofus.force.com/s/group/0F980000000CvlMCAS
New: https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
~- [ ] Any net new LWC work has JEST test coverage 50% or above~
~- [ ] Default Sa11y tests pass for all LWC components~
~- [ ] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary~
~- [ ] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
~- [ ] Make sure that ACs are updated (if any gaps)~
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [x] Code Reviewer
- [x] Pull Request contains draft release notes
~- [ ] Labels, help text, and customer facing messages are reviewed by Docs~
- [x] QE story level testing completed

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -136,7 +136,7 @@ plans:
         tier: primary
         is_listed: True
         preflight_message: "This will install Outbound Funds Module into your org."
-        post_install_message: "Thanks for installing Outbound Funds Module. Please visit the [Outbound Funds Module](https://powerofus.force.com/s/group/0F980000000CvlMCAS) community group on the Power of Us Hub for any questions about Outbound Funds Module."
+        post_install_message: "Thanks for installing Outbound Funds Module. Please visit the [Outbound Funds Module](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE) community group on the Power of Us Hub for any questions about Outbound Funds Module."
```
Comment (melissabarber) [inline cumulusci.yml]:
```suggestion
        post_install_message: "Thanks for installing Outbound Funds Module. Please visit the [Outbound Funds Module](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE) community group on the Trailblazer Community for any questions about Outbound Funds Module."
```

**Review by melissabarber — APPROVED** [review]
Updates here look good. In the WI for RelEng, please request for them to update the last sentence on the product installation page:

Questions? Become a member of the [Outbound Funds Module ](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE)community group on the Trailblazer Community.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The Outbound Funds app is a community developed and maintained Open Source Commo
 Links:
 
 -   [Documentation/Wiki](https://github.com/SalesforceFoundation/OutboundFundsModule/wiki)
--   [Power of Us Hub group](https://powerofus.force.com/s/group/0F980000000CvlMCAS/community-project-outbound-funds)
+-   [Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE/community-project-outbound-funds)
```
Comment (melissabarber) [inline README.md:12]:
```suggestion
-   [Outbound Funds group on Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE)
```

**Review by melissabarber — COMMENTED** [review]
@scottwarren-sfdo left one suggested change to fix the URL and add context to where the URL is pointing to.

File: README.md (hunk 2 above)
Comment (scottwarren-sfdo) [reply README.md]: I caught this as well and pushed a change a little while ago. Resolving this comment.

**Review by jstvz — APPROVED** [review]
CODEOWNERS 🆗

**Review by spelak-salesforce — APPROVED** [review]
Changes look accurate and complete for all translations.

**Pull request merged**